cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -151,6 +155,9 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 				)
 			}
 		}
+		if cStorVolumeGot.Status.Capacity.IsZero() {
```

Comment:
this check and set looks to be a hack.. it need to be moved into `CreateVolumeTarget` or at the success of `CreateVolumeTarget`

Reply:
removed and updated when CreateVolumeTarget is successful.